winit-core/src/keyboard.rs

```diff
@@ -1701,6 +1701,8 @@ bitflags! {
         const ALT = 0b100 << 6;
         /// This is the "windows" key on PC and "command" key on Mac.
         const META = 0b100 << 9;
+        /// The "AltGraph" key, usually used to insert symbols.
+        const ALTGR = 0b100 << 10;
```

Comment:
```suggestion
        const ALT_GRAPH = 0b100 << 10;
```

To match [`keyboard_types::Modifiers::ALT_GRAPH`](https://docs.rs/keyboard-types/0.8.0/keyboard_types/struct.Modifiers.html#associatedconstant.ALT_GRAPH).

Reply:
renamed

Reply:
With an underscore please ;)

Reply:
Oh, sorry! Glazed right over it